src/components/TextWithDivider/index.css

```diff
@@ -0,0 +1,20 @@
+.text-divider {
+  display: flex;
+  align-items: center;
+}
+
+.text-divider::before,
+.text-divider::after {
+  content: "";
+  height: 1px;
+  background-color: #eee;
+  flex-grow: 1;
+}
+
+.text-divider::before {
+  margin-right: 5px;
+}
+
+.text-divider::after {
+  margin-left: 5px;
+}
```

Comment:
why not use tailwind?

Reply:
I am not aware of the classes which work similar to ::before & ::after classes. If I find any alternative I will work on removing this stuff in follow up PR. 

Can we you please create a doc around the coding guidelines and link it in the README.md file, that will give us an idea what is allowed and what is not.